contrib/IECoreArnold/src/IECoreArnold/UniverseBlock.cpp

```diff
@@ -53,7 +53,12 @@ namespace
 void loadMetadata( const std::string &pluginPaths )
 {
 	typedef boost::tokenizer<boost::char_separator<char> > Tokenizer;
-	Tokenizer t( pluginPaths, boost::char_separator<char>( ":" ) );
+	#ifdef _WIN32
+		const std::string separator = ";";
```

Comment:
There's no need to allocate a new string here, since we end up using `separator.c_str()` anyway. We can simply use `const char *separator = ...`.